app.js

```diff
@@ -9,3 +11,18 @@ const signUpPassword = document.getElementById('sign-up-password');
 // Wire up sign in and sign up forms to supabase
 // Redirect to /other-page on successful auth
 // Redirect to /other-page when page loads if user is authenticated
+
+signUpForm.addEventListener('submit', async (e) => {
+    e.preventDefault();
+    //console.log(signUpEmail.value, signUpPassword.value);
```

Comment:
Nice work here. Here is another way you could have written it. `    const formData = new FormData(signInForm);
    const user = {
        email: formData.get('email'),
        password: formData.get('password')
    };
    await signInUser(user.email, user.password);`